indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/TaskMonitor.java

```diff
@@ -84,6 +85,11 @@
   private int numRunningTasks;
   private int numSucceededTasks;
   private int numFailedTasks;
+  // This metric is used only for unit tests because the current taskStatus system doesn't track the killed task status.
+  // Currently, this metric only represents # of killed tasks by ParallelIndexTaskRunner.
+  // See killAllRunningTasks(), SinglePhaseParallelIndexTaskRunner.run(), and
```

Comment:
There was no method `killAllRunningTasks()` anywhere in the repo when this PR was checked in.

Reply:
Oops, thanks. Will fix it soon.

Reply:
@leventov fixed in https://github.com/apache/incubator-druid/pull/8924.